CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.1] - 2023-08-29
```

Comment:
"It's not a bug, it's a feature" - literally. Let's make it 2.1.0 :)

Reply:
Version updated to 2.1.0